doc/RecordingControl.xml

```diff
@@ -1684,8 +1766,212 @@ SetRecordingJobMode(JobToken, Active)
+        <literal>time</literal> is not conformant to the extended time format defined in ISO 8601-1, as it replaces <literal>":"</literal> with <literal>"-"</literal> compared to that format.
+      </para>
+    </section>
+    <section>
+      <title>Span end objects</title>
+      <para>
+        To store the end time of of a span, the device should store an additional empty object after closing a span.
+        The device shall generate the object key for this object according to the ABNF <literal>key</literal> rule and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the span end in Universal Coordinated Time (UTC), following the ABNF rules.
+        The device shall replace <literal>counter</literal> by a value equal to the counter of the last segments of the span plus 1.
+        The device shall replace <literal>extension</literal> with a value defined by the recording formats in the following sections.
+      </para>
+    </section>
+    <section>
+      <title>Other objects</title>
+      <para>
+        The device may store additional objects with object keys starting with <literal>lead "/"</literal> according to the ABNF rules and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />, as long as their object keys do not end in an extension defined in this annex.
+      </para>
+    </section>
+    <section>
+      <title>Recording format MP4</title>
+      <para>
+        For the recording format <literal>MP4</literal>, the recording information shall be written as multiplexed tracks into a single segment.
+        Each segment shall contain zero or one tracks with video data, zero or one tracks with audio data, and zero or one tracks with metadata.
+        The device shall generate the object key for each segment with the <literal>extension</literal> <literal>"mp4"</literal>.
+      </para>
+      <para>
+        The device shall upload each segment object with the content type <literal>video/mp4</literal>.
+        The device should include the <literal>codecs</literal> parameter in the content type according to RFC 6381.
+      </para>
+      <para>
+        The <literal>MovieBox</literal> shall not reference any samples.
+      </para>
+      <para>
+        A segment shall contain a <literal>MovieFragmentRandomAccessBox</literal> as the last box.
+        The <literal>MovieFragmentRandomAccessBox</literal> shall provide references for all tracks and their applicable <literal>MovieFragmentBox</literal> instances.
+      </para>
+      <para>
+        The device shall upload a span end object.
+        It should generate the object key for a span object with the <literal>"mp4_end"</literal> as the <literal>extension</literal>.
+      </para>
+    </section>
+    <section>
+      <title>Recording format CMAF</title>
```

Comment:
Open point: Do we want to mandate that the segments contain a `MovieFragmentRandomAccessBox` for the `CMAF` format? Is this compatible with the CMAF specification?

Reply:
As discussed in Singapore, this will not be mandated for `CMAF`, but requirements for an optional `MovieFragmentRandomAccessBox` will be added.